### PR TITLE
Remove highlights

### PR DESCRIPTION
**Pull request by paususe**
This PR removes the yellow boxes we have in several pages.

The reason for removing them is those yellow boxes are too prominent and bring all the attention, so e. g. the release notes for Uyuni Stable are very easy to miss (happens even to me!).

While doing that, I have also fixed some typos, language and removed outdated text (e. g. we no longer need to have a warning about updating from 2020.01: it's been more than 1 year already and Uyuni is rolling release, and that warning was also in the release notes).

### CODE REVIEW

File: pages/contact.html [hunk 1]
```diff
@@ -99,7 +99,7 @@ <h4><a href="#ml">Mailing lists</a></h4>
             </ul>
             <h4><a href="#bugs">Reporting bugs</a></h4>
             <p> Submit bugs to the issues tab located at our <a href="https://github.com/uyuni-project/uyuni/issues">Github repository</a>.</p>
```
Comment (Loquacity) [inline pages/contact.html:101]:
```suggestion
            <p>Submit bugs using the issues tab in the <a href="https://github.com/uyuni-project/uyuni/issues">GitHub repository</a>.</p>
```

File: pages/contact.html [hunk 2]
```diff
@@ -99,7 +99,7 @@ <h4><a href="#ml">Mailing lists</a></h4>
...
-            <div class="alert alert-warning" role="alert"> Verify that a duplicate of the issue does not already exist! </div>
+            <p>Please verify that a duplicate of the issue does not already exist!</p>
             <h4><a href="#community-hours">Uyuni Community Hours</a></h4>
             <p>Every last Friday of the month, 16.00 CET/CEST (Germany time) we meet online to talk about what is new in Uyuni, discuss how to improve and enhance it, answer questions from the community and have a nice time in general.</p>
```
Comment (Loquacity) [inline pages/contact.html:104]:
```suggestion
            <p>Every last Friday of the month, at 16.00 CET/CEST (Germany time) we meet online to talk about what is new in Uyuni, discuss how to improve and enhance it, answer questions from the community, and have a nice time in general.</p>
```

File: pages/devel-version.html [hunk 3]
```diff
@@ -92,8 +92,8 @@ <h4><a href="#warning">Warning</a></h4>
             <p>Never use the Uyuni Development Version for production systems in any way!</p>
             <p>Uyuni Development Version is updated with each commit to the Git Repository, and it has the following know limitations:</p>
             <ul>
-                <li>It can break at any moment</li>
-                <li>Upating to newer development versions does not work</li>
+                <li>It may break at any moment</li>
```
Comment (Loquacity) [inline pages/devel-version.html:95]: Why 'may' instead of 'can'?

File: pages/devel-version.html [hunk 4]
```diff
@@ -257,10 +257,8 @@ <h4>Client Tools</h4>
             </div>
             
             <h4>Reporting Bugs</h4>
-            <p> Submit bugs to the issues tab located at our <a href="https://github.com/uyuni-project/uyuni/issues">Github repository</a>.</p>
-
-            <div class="alert alert-warning" role="alert">
-                Verify that a duplicate of the issue does not already exist! </div>
+            <p>Submit bugs to the issues tab located at our <a href="https://github.com/uyuni-project/uyuni/issues">Github repository</a>.</p>
```
Comment (Loquacity) [inline pages/devel-version.html:260]:
```suggestion
            <p>Submit bugs using the issues tab at our <a href="https://github.com/uyuni-project/uyuni/issues">GitHub repository</a>.</p>
```

File: pages/devel-version.html [hunk 5]
```diff
@@ -257,10 +257,8 @@ <h4>Client Tools</h4>
...
+            <p>Please verify that a duplicate of the issue does not already exist.</div>
```
Comment (Loquacity) [inline pages/devel-version.html:261]:
```suggestion
            <p>Please verify that a duplicate of the issue does not already exist!</div>
```

File: pages/stable-version.html [hunk 6]
```diff
@@ -93,13 +93,14 @@ <h4><a href="#releasenotes">Release notes</a></h4>
                 <li><a href="../doc/2021.02/release-notes-uyuni-server.html">Server</a></li>
                 <li><a href="../doc/2021.02/release-notes-uyuni-proxy.html">Proxy</a></li>
             </ul>
-            <div class="alert alert-warning" role="alert">Remember to review the release notes! They contain value information and, depending on your current version, required procedures to update to the most recent one!</div>
-	    <div class="alert alert-warning" role="alert">If you are not updating from 2021.01 and if the WebUI is not starting and shows a HTTP 404 after the update, apply this <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/message/OHWHT3XBN6DX5NMTTADAVF4GKPMWQ6LZ/" target="_blank">fix</a>. This information is available at the <a href="../pages/stable-version.html#releasenotes">release notes</a> as well.</div>
+            <p>Remember to review the release notes! They contain value information and, depending on your current version, required procedures to update to the most recent one!</p>
```
Comment (Loquacity) [inline pages/stable-version.html:96]:
```suggestion
            <p>Remember to review the release notes! They contain valuable information and, depending on your current version, required procedures to update to the most recent version.</p>
```

File: pages/stable-version.html [hunk 7]
```diff
@@ -274,10 +275,8 @@ <h4>Client Tools</h4>
             </div>
 
             <h4>Reporting Bugs</h4>
-            <p> Submit bugs to the issues tab located at our <a href="https://github.com/uyuni-project/uyuni/issues">Github repository</a>.</p>
-
-            <div class="alert alert-warning" role="alert">
-                Verify that a duplicate of the issue does not already exist! </div>
+            <p>Submit bugs to the issues tab located at our <a href="https://github.com/uyuni-project/uyuni/issues">Github repository</a>.</p>
```
Comment (Loquacity) [inline pages/stable-version.html:278]:
```suggestion
            <p>Submit bugs using the issues tab at our <a href="https://github.com/uyuni-project/uyuni/issues">GitHub repository</a>.</p>
```

File: pages/devel-version.html (hunk 3 above)
Comment (paususe) [reply pages/devel-version.html:95]: In my non-native-English mind, that expressed probability. But you are the native speaker, so I guess "can" is the right wording :-)

Comment (Loquacity) [reply pages/devel-version.html:95]: Both are correct, but 'may' is a little more ambiguous (it means that you have the ability to do something, or are choosing whether or not to do something, or that you have permission to do something), so most tech writers prefer 'can'. It mostly comes down to personal preference, though. I was mostly just curious :wink:

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
As I said at the other PR: While I agree with doing the that notes less prominent, I don't agree with downgrading them to a simple paragraph, so please propose an alternative to the yellow boxes. Mine is using `<strong>WARNING</strong>` or `<strong>IMPORTANT</strong>`

We really need users to pay attention to those texts, so they need some kind of highlight.

@Loquacity any advice?

**Comment by Loquacity** [comment]
> As I said at the other PR: While I agree with doing the that notes less prominent, I don't agree with downgrading them to a simple paragraph, so please propose an alternative to the yellow boxes. Mine is using `<strong>WARNING</strong>` or `<strong>IMPORTANT</strong>`
> 
> We really need users to pay attention to those texts, so they need some kind of highlight.
> 
> @Loquacity any advice?

They look like you're trying to mimic what we would call an admonition in the docs. Frankly, the things we have called out here, if we had them in the docs, would not warrant being put in an admonition. It's mostly just general information that relates to the previous sentence. So if you want to keep them in a box, make the background colour lighter and cooler, such as a pale green or blue. Or keep them as plain text, but use the additional punctuation (exclamation marks, etc, but don't overdo it), or some extra words at the beginning of the sentence to draw the eye, like "It is important that..." or "Make sure you ...".

**Comment by juliogonzalez** [comment]
What about `<p><strong>IMPORTANT:</strong> blablabla</p>?

**Comment by paususe** [comment]
> What about `<p><strong>IMPORTANT:</strong> blablabla</p>`?

A bit better but still the same problem as with the yellow boxes: reader's eyes immediately go to the bold text and reader will miss the release notes, which are above it. Normal text will work just fine, we do not need yellow boxes, bold faces or anything like that.

image

image

**Comment by juliogonzalez** [comment]
What yellow boxes? My suggestion is having things like in your second screenshot, with the bold **IMPORTANT** text before the important texts.

**Comment by paususe** [comment]
> What yellow boxes? My suggestion is having things like in your second screenshot, with the bold **IMPORTANT** text before the important texts.

The current yellow boxes:
image

IMHO we do not need the yellow boxes or the bold **IMPORTANT**. It's only distracting the reader from seein the release notes.

**Comment by juliogonzalez** [comment]
I was suggesting to replace the yellow boxes with something less prominent such as the bold **IMPORTANT** In front of the important texts.

Sorry to disagree, but even with the links, people is not reading the release notes, so I want to keep some kind highlight. Same for the other pages.

**Comment by paususe** [comment]
> even with the links, people is not reading the release notes

Exactly: people are not reading the release notes because they are distracted by the highlighted text so they miss the other page. Even I had not noticed we were doing release notes for Uyuni until a few months ago (!).

>, so I want to keep some kind highlight. Same for the other pages.

Persisting the highlighting is persisting the problem which makes people not read the release notes.

Let's try without any kind of highlight or bold type faces and see what happens. Can't be worse than now.

**Comment by juliogonzalez** [comment]
> Let's try without any kind of highlight or bold type faces and see what happens. Can't be worse than now.

No, I don't agree. The very same reason the highlighting was added is because we had a lot of questions that were part of the release notes. And since then we had less problems, and no more.

So again: I agree that the yellow box is maybe too much, but I don't agree with keeping that as a simple paragraph. At the very least add **IMPORTANT:**.

File: pages/stable-version.html [hunk 8]
```diff
@@ -93,13 +93,14 @@ <h4><a href="#releasenotes">Release notes</a></h4>
                 <li><a href="../doc/2021.02/release-notes-uyuni-server.html">Server</a></li>
```
Comment (juliogonzalez) [inline pages/stable-version.html:93]: Maybe you can consider to make this more prominent, while you keep the **Important** without using capital letters. That  could be a good comprimise.

For example:
```
<ul>
    <li><a href="../doc/2021.02/release-notes-uyuni-server.html">For Uyuni Server</a></li>
    <li><a href="../doc/2021.02/release-notes-uyuni-proxy.html">For Uyuni Proxy</a></li>
<li>
<p><strong>Important:</string>Remember to review the release notes! They contain valuable information and, depending on your current version, required procedures to update to the most recent version.</p>
```

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

**Comment by juliogonzalez** [comment]
Closing, as in the end we got rid of the "yellow" boxes:

image

**Pull request closed without merging**